RR.Tests.Data/Models/HRIS/EmployeeQualificationTestData.cs

```diff
@@ -0,0 +1,39 @@
+﻿using HRIS.Models;
+using HRIS.Models.Enums.QualificationEnums;
+
+namespace RGO.Tests.Data.Models;
+
+public static class EmployeeQualificationTestData
+{
+    //public static EmployeeQualificationDto BachelorQualification => new EmployeeQualificationDto
+    //{
+    //    Id = 1,
+    //    EmployeeId = 1,
+    //    Qualification = "Bachelor's Degree",
+    //    School = "University of Pretoria",
+    //    Degree = (DegreeType)Enum.Parse(typeof(DegreeType), "Bachelor"),
```

Comment:
Ghost code?

Reply:
Its for the Unit tests. Will remove it asap